nemo/collections/multimodal/speech_llm/models/modular_s2s_models.py

```diff
@@ -816,7 +806,7 @@ def write_predictions_to_file(self, outputs, output_file_path_prefix, output_dir
     def de_concat_multiproj_logits(self, logits):
         logits_list = []
         prev = 0
-        for i in self.model.proj_head_dims:
+        for i in self.model.module.proj_head_dims:
```

Comment:
same here.
can u confirm you change does not break amp=False case

Reply:
confirmed